### DISCUSSION TITLE
Student package not ready still after 4 days

### DISCUSSION THREAD

**Discussion by NoaHalali** [community/community#136635] [GitHub Education]
### Hi everyone,

i'm trying to activate my github education that i've got free from my college.
I saw the messege:
Congrats! Your request for [@NoaHalali](https://github.com/NoaHalali) was approved and your benefits will be available [soon](https://github.com/orgs/community/discussions/111352#user-content-how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

but still when i'm log in to thew github education I see that my package will be ready soon

**Comment by davevad93** [comment]
Hi @NoaHalali ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @NoaHalali was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
